private/buf/cmd/buf/command/beta/registry/commit/commitget/commitget.go

```diff
@@ -38,7 +38,7 @@ func NewCommand(
 	flags := newFlags()
 	return &appcmd.Command{
 		Use:   name + " <buf.build/owner/repo:ref>",
-		Short: "Get commit details",
+		Short: "Get details about a commit.",
```

Comment:
nit: From above, we're changing to the `Creates ...` tense, but this is `Get`. Should it be `Gets ...` and `Creates ...` or `Get ...` and `Create ...`?

The same could be said of other commands, like `List ...` vs. `Lists ...`. I kinda like the singular form, but I'm open to your preference as long as it's consistent (I don't know how to refer to these tenses).

Reply:
Good call on the discrepancies. My experience with other recent CLI tools suggests that it should be Create, Get, etc. That's not a scientific viewpoint, but let's standardize on that for now.